recipes/emsarray/meta.yaml

```
@@ -0,0 +1,76 @@
+{% set name = "emsarray" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/emsarray-{{ version }}.tar.gz
+  sha256: 63c92c95483ec263aecf88266acf102513e2ce74099c035172affe4eff959729
+
+build:
+  number: 0
+
+outputs:
+  - name: emsarray-core
+    build:
+      noarch: python
+      script: python -m pip install . -vv --no-deps
+      entry_points:
+        - emsarray = emsarray.cli:main
+
+    requirements:
+      host:
+        - python >=3.8
+        - pip
+      run:
+        - python >=3.8
+        - bottleneck >=1.3
+        - geojson >=2.5.0
+        - netcdf4 >=1.5.3
+        - numpy >=1.18.0
+        - shapely >=1.8.0
+        - xarray >=0.18.2
+    test:
+      imports:
+        - emsarray
+      commands:
+        - pip check
+        - emsarray --help
+      requires:
+        - pip
+
+  - name: emsarray
+    build:
+      noarch: python
+    requirements:
+      host:
+        - python >=3.8
+        - pip
+      run:
+        - {{ pin_subpackage("emsarray-core", exact=True) }}
+          # plot extra
+        - matplotlib-base >=3.4.3
+        - scipy >=1.6.0
+        - cartopy >=0.20.0
+        - pykdtree >=1.2.2
+          # tutorial extra
+        - pooch >=1.3.0
+    test:
+      imports:
+        - emsarray
+
+
+about:
+  home: https://pypi.org/project/emsarray/
+  summary: Utilities for working with non-CF grid datasets
+  license: LicenseRef-CSIRO-Open-Source-Software-Licence-Agreement
+  license_family: BSD
+  license_file: LICENCE
+  doc_url: https://emsarray.readthedocs.io/
+  dev_url: https://github.com/csiro-coasts/emsarray/
+
+extra:
+  recipe-maintainers:
+    - mx-moth
```

```suggestion
    - mx-moth
  feedstock-name: {{ name|lower }}
```
This makes sure that the right name is chosen for the feedstock.